packages/codebytes/package.json

```diff
@@ -23,12 +23,15 @@
     "@codecademy/gamut": "*",
     "@codecademy/gamut-icons": "*",
     "@codecademy/gamut-styles": "*",
+    "@codecademy/tracking": "0.18.0",
     "@codecademy/variance": "*",
     "@emotion/react": "^11.4.0",
     "@emotion/styled": "^11.3.0",
     "@monaco-editor/react": "4.3.1",
     "monaco-editor": ">= 0.25.0 < 1",
-    "react-resize-observer": "1.1.1"
+    "react-resize-observer": "1.1.1",
+    "js-base64": "^3.6.0",
+    "jsuri": "^1.3.1"
```

Comment:
see https://github.com/Codecademy/client-modules/pull/19/files#r793053273 about how we can remove this dependency